test/testHelpers.hpp

```diff
@@ -530,6 +541,25 @@ af::dim4 unravelIdx(uint idx, af::array arr) {
 struct FloatTag {};
 struct IntegerTag {};
 
+template<typename T>
+std::string printContext(const std::vector<T>& hArr, af::dim4 arrStrides,
+                         af::dim4 tgtCoords) {
+    int idx = ravelIdx(tgtCoords, arrStrides);
+    int ctxWidth = 5;
+    int startIdx = std::max<int>(0, idx - ctxWidth);
+    int endIdx = std::min<int>(idx + ctxWidth + 1, hArr.size());
+    std::ostringstream os;
+    os << "{ ";
+    for (uint i = startIdx; i < endIdx; ++i) {
+        if (i == idx)
+            os << "[" << hArr[i] << "]" << " ";
```

Comment:
I think this would cause the inputs to be unaligned after the value. Can you try to keep alignment by printing empty characters around the gold array?

Reply:
Never mind. Looks like you are wrapping both gold and actual values. That is fine.